### DISCUSSION TITLE
.node-version no longer updating in sync with .gitlab-ci

### DISCUSSION THREAD

**Discussion by leaumar** [renovatebot/renovate#9753] [Request Help]
I have the following config:
```json
{
    "extends": ["config:base", ":automergeBranch", ":automergePatch", ":automergeTypes", ":pinVersions"],
    "nvm": {
        "fileMatch": ["^\\.node-version$"]
    }
}
```

This used to lead to PRs where both my .gitlab-ci and .node-version file get the same update at the same time once the version is live on both (possibly needing some re-runs for both to be released):
![image](https://user-images.githubusercontent.com/3950300/116299186-a717f000-a79d-11eb-957b-472a80c1f09d.png)

The last time I got such a PR was 2 months ago. Since then, I've been getting PRs for node 15 and now even 16, but they never include .node-version, only .gitlab-ci. Renovate seems to still be matching the file, just not pushing for an update to it:
```json
{
  "managers": {
	"gitlabci": {"fileCount": 1, "depCount": 1},
	"nodenv": {"fileCount": 1, "depCount": 1},
	"npm": {"fileCount": 1, "depCount": 22},
	"nvm": {"fileCount": 1, "depCount": 1}
  },
  "total": {"fileCount": 4, "depCount": 25}
}
```

Is this no longer the correct config to keep .node-version and .gitlab-ci updated together? I started getting PRs for the [alpine image](https://github.com/nodejs/docker-node/blob/f0551c9eb5e0dbb853eda4266bc0964e5989bda1/16/alpine3.13/Dockerfile) released 5d if I'm not mistaken since this morning, but the [node version itself](https://nodejs.org/en/about/releases/) has been out a week already. This is the only PR about node versions in every repo, .node-version isn't being updated separately or anything either.
![image](https://user-images.githubusercontent.com/3950300/116300469-396cc380-a79f-11eb-9bc4-aa32ac61cdd3.png)

**Comment by rarkins** [comment]
Please look for the `packageFiles with updates` log message (debug level) and look for these two dependencies in particular to see what Renovate reports. e.g. does it detect any new versions, etc. 

I suspect it might be because one of them uses our `node` versioning - which knows that v15 and v16 are unstable and should not be proposed - while that's perhaps not happening for the node Docker image.

**Comment by leaumar** [comment]
Ah yeah, that would explain it. Are we talking about the start of LTS (in october) or those stats that renovate now gathers about build stability? I'll debug-run later.

**Reply by rarkins** [reply]
We added a *feature* in #7157  of detecting unstable node versions to hold back undesirable stable->unstable upgrade proposals, such as 14->15 and 14->16.

But if you have recently upgraded to Renovate v25, it's possibly this which has affected you:
![image](https://user-images.githubusercontent.com/6311784/116308336-a8e6b100-a7a7-11eb-827b-fdf4eea14a4d.png)
